pkgs/development/libraries/glibmm/default.nix

```
@@ -11,7 +11,10 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  nativeBuildInputs = [ pkgconfig gnum4 ];
+
+  #  While compiling, glibmm tries to execute glib-compile-schemas, which
+  #  requires glib to be in the nativeBuildInputs for cross-platform builds.
+  nativeBuildInputs = [ pkgconfig gnum4 glib ];
```

This is too verbose and non-local for my taste, something like this would be better IMO:

```suggestion

  nativeBuildInputs = [
    pkgconfig
    gnum4
    glib #  for glib-compile-schemas
  ];
```

---

done